src/App.tsx

```diff
@@ -1,17 +1,103 @@
-import React from 'react';
-import './App.css';
-
+import React, { useState } from 'react';
 import users from './api/users';
+import './App.css';
+import TodoList from './components/TodoList/TodoList';
+import { preparedTodos } from './types/Preraredtodo';
 
 const App: React.FC = () => {
+  const [title, SetTitle] = useState('');
+  const [user, SetUser] = useState('');
+  const [isValidTitle, SetValidTitle] = useState('');
```

Comment:
why capital letter at the start? it's not a class, fix it everywhere
```suggestion
  const [isValidTitle, setValidTitle] = useState('');
```